luigi/lock.py

```diff
@@ -33,9 +33,12 @@ def getpcmd(pid):
 
     :param pid:
     """
-    cmd = 'ps -p %s -o command=' % (pid,)
+    cmd = 'ps -o pid,comm | grep %s | xargs -n 2' % (pid,)
```

Comment:
And yes, I originally thought of fully parsing in python. I might do that for easy of testability etc.